docs/openj9_support.md

```diff
@@ -172,9 +172,9 @@ OpenJDK 17 binaries are expected to function on the minimum operating system lev
 | Windows Server 2016                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 | Windows Server 2019                       | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
 
-| macOS                                     | x64                                                                                  |
-|-------------------------------------------|--------------------------------------------------------------------------------------|
-| OS X 10.14+                               | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| macOS         | x64                                                                                  | AArch64                                                                               |
+|---------------|--------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| OS X 10.14+   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
```

Comment:
The table for OpenJDK 17 on macOS should look like this.

```
|--------------|-----|---------------|
| macOS        | x64 | Apple Silicon |
|--------------|-----|---------------|
| macOS 10.14+ | v   | N/A           |
| macOS 11     | v   | v             |
| macOS 12     | v   | v             |
|--------------|-----|---------------|
```

"OS X" is the old name for the versions before macOS Sierra (10.12).